Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -149,6 +149,8 @@ void ParticleHistogram2D::ComputeDiags (int step)
     }
 
     d_data_2D.copy(m_h_data_2D);
+    Gpu::streamSynchronize();
```

Comment:
This sync is not needed, because there is a sync in WarpXParIter constructor later.